docs/cli-command-reference.md

```
@@ -1,360 +1,494 @@
+docsfy --host docsfy.example.com --port 443 -u admin -p <API_KEY> health
+```
 
-`docsfy config` manages the CLI config file at `~/.config/docsfy/config.toml`.
+Return value / effect:
+- Dispatches to `health`, `config`, `generate`, `list`, `status`, `delete`, `abort`, `download`, `models`, or `admin`.
+- With no subcommand, prints help.
+- Applies the resolved connection settings to the selected subcommand.
 
-A sample config from the repository:
+> **Note:** Connection resolution order is explicit CLI flags, then `--server`, then `[default].server` in `~/.config/docsfy/config.toml`, then an error if nothing is configured.
 
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)